src/bzmesh.jl

```diff
@@ -44,8 +44,6 @@ given the crystal symmetries `symmetries`. Returns the list of irreducible ``k``
 the grid.
 """
 function bzmesh_ir_wedge(kgrid_size, symmetries; kshift=[0, 0, 0])
-    all(isequal.(kgrid_size, 1)) && return bzmesh_uniform(kgrid_size; kshift)
-
```

Comment:
Why? This actually helps in many cases with special floating-point types.

Reply:
I did this because bzmesh_uniform does not know about floating point types and therefore returns symmetries as boolean type, which causes trouble later. We should probably refactor these so these functions don't return symmetries and just filter them later, anyway

Reply:
Does not seem to be a problem any more. What was the issue exactly?